cli/policy/policy.go

```diff
@@ -54,9 +54,12 @@ func Update(fileName string) {
 }
 
 func Patch(patch string) {
+	msgPrinter := i18n.GetMessagePrinter()
+	msgPrinter.Printf("Warning: This command is deprecated. It will still be working until the next major release. Please use 'hzn policy update' command to update the node policy in the future.")
```

Comment:
Please change the message to:
"Warning: This command is deprecated. It will continue to be supported until the next major release. Please use 'hzn policy update' to update the node policy."

Reply:
done